src/ModalManager.js

```diff
@@ -20,21 +20,48 @@ function findContainer(data, modal) {
     d => d.modals.indexOf(modal) !== -1);
 }
 
+function setContainerStyle(state, container) {
+  let style = { overflow: 'hidden' };
+
+  // we are only interested in the actual `style` here
+  // becasue we will override it
+  state.style = {
+    overflow: container.style.overflow,
+    paddingRight: container.style.paddingRight
+  }
+
+  if (state.overflowing) {
+    // use computed style, here to get the real padding
+    // to add our scrollbar width
+    style.paddingRight =
+      parseInt(css(container, 'paddingRight') || 0, 10) + getScrollbarSize() + 'px';
+  }
+
+  css(container, style);
+}
+
+function removeContainerStyle({ style }, container) {
+
+  Object.keys(style).forEach(
```

Comment:
`Object.assign(container.style, style)`?

might need to pull in that Babel transform though